src/Logging/LoggerHelper.php

```diff
@@ -42,8 +45,13 @@ public static function logDelayableProcessFailException(DelayableProcessFailExce
     }
 
 
-    public static function logJobPushedIntoQueue(JobInterface $job, string $queueName, LoggerInterface $logger, ?JobType $jobType = null): void
-    {
+    public static function logJobPushedIntoQueue(
+        JobInterface $job,
+        string $queueName,
+        LoggerInterface $logger,
+        ?JobType $jobType = null,
+        int $delayInSeconds = self::NOT_DELAYED
```

Comment:
I would go for nullable parameter with `null` being "unknown". It must be unknown, because of backward compatibility - you can have a call of this method in your code and forget to change it after the upgrade.

```suggestion
        ?int $delayInSeconds = null
```

Then either put the `null` into the log context, or omit it entirely.

Reply:
This would mean that the two unchanged method calls should have a `0` added as the parameter value.

Reply:
I thought about null, but I want to be sure that elk mapping will create this field as numeric from beginning.